components/patient-access-rules/api/src/main/resources/ApplicationResources.properties

```diff
@@ -63,7 +63,7 @@ phenotips.accessRightsManagement.modifyUpdateFailure=Failed to update.
 phenotips.accessRightsManagement.modifyRemoveCollaborator=Remove this collaborator
 
 phenotips.accessRightsManagement.adminInfo=All administrator users have manage access to this record.
-phenotips.accessRightsManagement.guestRecordInfo=This record is created by guest user and can be seen by anyone.
+phenotips.accessRightsManagement.guestRecordInfo=This record is created by a guest user.
```

Comment:
It should probably stay as it was, the fact that not everybody can see the record is a bug which I'm fixing right now.